lib/tutorials/en_US/routing.md

```diff
@@ -65,63 +79,225 @@ server.route({
+Alternatively you may pass the same value to `h.response(value)` and return that from the handler. The result of this call is a response object, that can be chained with additional methods to alter the response before it is sent. For example `h.response('created').code(201)` will send a payload of created with an HTTP status code of 201. You may also set headers, content type,content length, send a redirection response, and many other things that are documented in the [API reference](/api#response-toolkit).
 
-## Config
+The `handler` option must return a value, a promise, or throw an error.
+
+Note: handlers using a fat arrow style function cannot be bound to any `bind` property. Instead, the bound context is available under [`h.context`](https://hapijs.com/api#h.context).
+
+## <a name="options" /> Options
 
 Aside from these three basic elements, you may also specify an `options` parameter for each route. This is where you configure things like [validation](/tutorials/validation), [authentication](/tutorials/auth), prerequisites, payload processing, and caching options. More details can be found in the linked tutorials, as well as the [API reference](/api#route-options).
 
-Here we will look at a couple of options designed to help generate documentation.
+Here we will look at some options of validating with Joi.
 
-```javascript
+```js
 server.route({
-    method: 'GET',
-    path: '/hello/{user?}',
+    method: 'POST',
+    path: '/signup',
     handler: function (request, h) {
 
-        const user = request.params.user ?
-            encodeURIComponent(request.params.user) :
-            'stranger';
-
-        return `Hello ${user}!`;
+        const payload = request.payload;
+        return `Welcome ${encodeURIComponent(payload.username)}!`;
     },
     options: {
-        description: 'Say hello!',
-        notes: 'The user parameter defaults to \'stranger\' if unspecified',
-        tags: ['api', 'greeting']
+        auth: false,
+        validate: {
+            payload: {
+                username: Joi.string().min(1).max(20),
+                password: Joi.string().min(7)
+            }
+        }
     }
 });
 ```
+The first property under `options` is `auth`.  `auth` will set the authentication configuration for the route.  Since this route is for a new user signing up, we will disable authentication.  
+
+The second property is `validate`.  This allows you to set validate rules for various request components, such as `headers`, `params`, `payload`, and `failAction`.  We use the Joi to validate the `request.payload`.  For more info, please check the [validation tutorial](/tutorials/validation).
+
+## <a name="missing" /> 404 Handling
```

Comment:
That's a good call but I wonder if it is in the right tutorial. In the whole tutorial we use `hapi` as an API server as we return JSON data from our endpoints. In that section, we use it as an application server rendering templates. I'm not sure such handling belong to an API server. That's definitely not a final call, I'm open to discussion. 😃

Reply:
Yeah, I see your point.  I can put it in the views tutorial, or even make an error handling tutorial.

Reply:
I fixed the request...left 404 there for now till we decide where it goes

Reply:
I'll ask on Slack for what other think of it and we'll see.

Reply:
Alright after some discussions on Slack, we decided that we'll keep the 404 handling in the routing tutorial but we'd rather not use `vision` in this tutorial. The way you wrote the 404 handling part atm is more suited for the Views tutorial so you can use it as is in that tutorial. Showcasing the catch-all routes pattern in the routes tutorial is a good addition but let's just simplify it by returning a response which is not HTML (custom string or something like that).

Reply:
Sounds good....will fix now

Reply:
Cool thank you. 👍